pkg/operator/vitesstopo/register_cells.go

```diff
@@ -0,0 +1,90 @@
+/*
+Copyright 2019 PlanetScale Inc.
```

Comment:
Oops I missed this earlier.

```suggestion
Copyright 2020 PlanetScale Inc.
```

Reply:
We should update this across the board then. I copied this from another file. All the copyrights are off right now. I'll correct this and then create another PR to update all the copyrights.

Reply:
Fixed.